bpf/go_nethttp.c

```diff
@@ -12,15 +12,19 @@
 
 #include "utils.h"
 #include "go_str.h"
+#include "go_byte_arr.h"
 
 // Tell the helpers which debug level variable to use
 #define DBG_LEVEL go_http_debug_level
 #include "bpf_dbg.h"
 
 char __license[] SEC("license") = "Dual MIT/GPL";
 
+#define EVENT_HTTP_REQUEST 1
+#define EVENT_GRPC_REQUEST 2
```

Comment:
I think that, for now, it's fine. But once we merge the PR and make sure everything is working, we could refactor a bit to provide two eBPF instrumenters separately (one for HTTP and another for GRPC, in different files), and communicate both with the userspace via separate ringbuffers, so we don't need to differentiate the event type.

Reply:
Yes I agree, I'll refactor this in a follow-up. After we successfully merge the goroutine PR.